### DISCUSSION TITLE
Cannot SSH into VM in console mode

### DISCUSSION THREAD

**Discussion by awlane** [utmapp/UTM#3927] [UTM Help (macOS)]
**Describe the issue**
Despite having port forwarding set up such that I can SSH into a VM in full graphics mode, I cannot SSH into it in console mode. I am aware the console mode does give me shell access, but I prefer the MacOS terminal for this use case and have no use for the GUI and are using it as a stopgap for headless mode. This is consistently reproducible by booting into console mode.
This is a "custom" Ubuntu Server 20.04.4 ARM64 VM with no software added beyond install. 

**Configuration**
* UTM Version: 3.1.5 (53)
* OS Version: MacOS Monterey (12.3.1)
* Intel or Apple Silicon? Apple Silicon

**Crash log**
N/A

**Debug log**
[debuglogqemu.log](https://github.com/utmapp/UTM/files/8544019/debuglogqemu.log)

**Upload VM**
[configplist.txt](https://github.com/utmapp/UTM/files/8544015/configplist.txt)

**Comment by osy** [comment]
As you are using emulated network, you also forwarded guest port 22 to host port 666. That means to connect to SSH, you need to connect to localhost at port 666.
